GoogleTestAdapter/Core.Tests/Runners/CommandLineGeneratorTests.cs

```diff
@@ -309,6 +309,27 @@ public void GetCommandLines_ManyTestsWithSuites_BreaksUpLongCommandLinesCorrectl
             splittedTestsAsSet.Should().BeEquivalentTo(testsAsSet);
         }
 
+    [TestMethod]
+    [TestCategory(Unit)]
+    public void GetCommandLines_HugeNumberOfSuitesToBeExecutedFully_WontBreakCommandLineCreation()
```

Comment:
Since this is absolutely crucial code for GTA, I feel we should add some more test cases (e.g., with/without user parameters; different execution cases, e.g. with/without a test not run by a suite (I think the "no suite" case should be covered by the other tests))... What do you think?

Reply:
What you propose sounds reasonable too me. However I don't have an overview about the current state of test-coverage. What you propose is, if I understand this correctly, independent of my changes?
I wrote the test (as you originally proposed) with the intention to show that without my changes the method will throw an exception (and not more).

Reply:
If you mean test coverage in the technical sense (which you probably don't), you can check this out via the CodeCov links (and it should also be shown on GitHub - if not, let me know). Reproducing the bug with a test is important, but since you have changed handling of suites right in the core of GTA, we should also make sure that the new code behaves correctly (and will not be broken in the future). However, ... (see below)